xbmc/platform/linux/input/LibInputSettings.cpp

```diff
@@ -0,0 +1,104 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "LibInputSettings.h"
+
+#include "FileItem.h"
+#include "utils/log.h"
+#include "ServiceBroker.h"
+#include "settings/lib/Setting.h"
+#include "utils/XBMCTinyXML.h"
+#if defined(HAVE_GBM)
+#include "windowing/gbm/WinSystemGbm.h"
+#elif defined(TARGET_RASPBERRY_PI)
+#include "windowing/rpi/WinSystemRpi.h"
+#elif defined(HAS_LIBAMCODEC)
+#include "windowing/amlogic/WinSystemAmlogic.h"
+#endif
+
+#include <algorithm>
+
+CLibInputSettings::CLibInputSettings()
+{
+  std::string xkbFile("/usr/share/X11/xkb/rules/base.xml");
```

Comment:
borrowing files from X11 when not using X11 is wrong

Reply:
we use `libxkbcommon` which needs `xkeyboard-config`, there is no way around this,